### PR TITLE
[cassandra] Add YAML for Cassandra version > 2.2

### PR DESCRIPTION
**Pull request by yannmh**
Thanks @johnaxel 

**Related PR**
https://github.com/DataDog/chef-datadog/pull/263

### CODE REVIEW

File: conf.d/cassandra.yaml.example [hunk 1]
```diff
@@ -21,54 +21,81 @@ init_config:
...
-          - UpdateInterval
-          - WriteCount
-          - WriteOperations
       exclude:
-        keyspace: system
+        keyspace:
+          - system
+          - system_auth
+          - system_distributed
+          - system_traces
```
Comment (sethrosenblum) [inline conf.d/cassandra.yaml.example]: Can we add OpsCenter to this list?  It's a very common tool and I don't think anyone cares about metrics on it.

Comment (yannmh) [inline conf.d/cassandra.yaml.example]: Thanks for your feedback on it @sethrosenblum. I am adding it to the list of exclusion.

**Comment by yannmh** [comment]
We are entirely dropping the old Cassandra YAML configuration file. @remh can you take a pass on this PR and merge it please ?

**Comment by remh** [comment]
Thanks!

**Pull request merged**